data.kidsfirstdrc.org/manifests/fence/fence-config-public.yaml

```diff
@@ -0,0 +1,146 @@
+APP_NAME: Gen3 Data Commons
+# Where fence microservice is deployed
+BASE_URL: https://data.kidsfirstdrc.org/user
+# flask's debug setting
+# WARNING: DO NOT ENABLE IN PRODUCTION
+DEBUG: false
+# if true, will automatically login a user with username "test"
+MOCK_AUTH: false
+# if true, will only fake a successful login response from Google in /login/google
+MOCK_GOOGLE_AUTH: true
```

Comment:
no!

Reply:
oh nvm, there is no Google login anyway. But let’s still change it to False just in case